tests/conftest.py

```diff
@@ -307,85 +290,281 @@ def simple_inventory_node() -> None:
+    title += "SOft coarse grained Monte-Carlo Acceleration (SOMA)"
+
+    reference = cript.Reference(
+        "journal_article",
+        title=title,
+        authors=["Ludwig Schneider", "Marcus Müller"],
+        journal="Computer Physics Communications",
+        publisher="Elsevier",
+        year=2019,
+        pages=[463, 476],
+        doi="10.1016/j.cpc.2018.08.011",
+        issn="0010-4655",
+        website="https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    )
+    return reference
+
+
+@pytest.fixture(scope="function")
+def simple_reference_dict() -> dict:
+    ret_dict = {
+        "node": ["Reference"],
+        "type": "journal_article",
+        "title": "Multi-architecture Monte-Carlo (MC) simulation of soft coarse-grained polymeric materials: SOft coarse grained Monte-Carlo Acceleration (SOMA)",
+        "authors": ["Ludwig Schneider", "Marcus Müller"],
+        "journal": "Computer Physics Communications",
+        "publisher": "Elsevier",
+        "year": 2019,
+        "pages": [463, 476],
+        "doi": "10.1016/j.cpc.2018.08.011",
+        "issn": "0010-4655",
+        "website": "https://www.sciencedirect.com/science/article/pii/S0010465518303072",
+    }
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_citation_node(simple_reference_node) -> cript.Citation:
-    """
-    minimal citation node
-    """
-    my_citation = cript.Citation(type="derived_from", reference=simple_reference_node)
+    citation = cript.Citation("reference", simple_reference_node)
+    return citation
+
 
-    return my_citation
+@pytest.fixture(scope="function")
+def simple_citation_dict(simple_reference_dict) -> dict:
+    ret_dict = {"node": ["Citation"], "reference": simple_reference_dict, "type": "reference"}
+    return ret_dict
 
 
 @pytest.fixture(scope="function")
 def simple_quantity_node() -> cript.Quantity:
-    """
-    minimal quantity node
-    """
-    my_quantity = cript.Quantity(key="mass", value=1.23, unit="gram")
+    quantity = cript.Quantity("mass", 11.2, "kg", 0.2, "std")
+    return quantity
 
-    return my_quantity
+
+@pytest.fixture(scope="function")
+def simple_quantity_dict() -> dict:
+    ret_dict = {"node": ["Quantity"], "key": "mass", "value": 11.2, "unit": "kg", "uncertainty": 0.2, "uncertainty_type": "std"}
+    return ret_dict
+
+
+@pytest.fixture(scope="function")
+def simple_software_node() -> cript.Software:
+    software = cript.Software("SOMA", "0.7.0", "https://gitlab.com/InnocentBug/SOMA")
+    return software
+
+
+@pytest.fixture(scope="function")
+def simple_software_dict() -> dict:
+    ret_dict = {"node": ["Software"], "name": "SOMA", "version": "0.7.0", "source": "https://gitlab.com/InnocentBug/SOMA"}
+    return ret_dict
+
+
+@pytest.fixture(scope="function")
+def simple_property_node(simple_material_node, simple_condition_node, simple_citation_node, simple_data_node, simple_process_node, simple_computation_node):
```

Comment:
this fixtuer says that it is a `simple property node` meaning that it has the bare required arguments and is the smallest node that it can be, however, this node has all the possible arguments for it, this seems more like `complex_property_node`


my naming convention for fixtures so far has been:
* `simple_node`: are nodes with just the required arguments and nothing more keeping them very simple
* `complex_nodes`: are nodes with all possible required arguments making them as big as possible

Reply:
I didn't not know your nomenclature.
I can rename them.
Actually all of my fixtures use as many attributes as possible, because that way I cover more of the code.

Reply:
yeah I was doing that at first too, but I think somewhere along the testing I came to the conclusion that if I am putting everything it is harder to test what is failing and trace the source of the bug, so I started doing both small tests and big tests where I thought it was clear to me. Let me know if I am not making any sense. 
I could also be wrong as well, feel free to let me know

I should probably document the `simple` and `complex` node convention somewhere so it is clear because it probably wouldn't be as clear for a new contributor just joining

Reply:
I renamed the nodes accordingly.
There is a lot more complex more then before.

Reply:
> > at some point we should split up our `conftest.py` into:
> > ```
> > * `fixtures/primary_nodes/`
> > 
> > * `fixtures/subobjects/`
> > 
> > * `fixtures/supporting_nodes/`
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > and maybe it might be a good idea to have
> > ```
> > * `fixtures/primary_nodes/simple_nodes,py`
> > 
> > * `fixtures/primary_nodes/simple_nodes_dict,py`
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > so that our conftest.py doesn't get unruly and become too big
> 
> Kind of done. I didn't seperate dicts and nodes though. I think it more naturally to maintain them at the same place.

NICE!!!

I did not see that, but I just checked it out and saw that you are already ahead of the game! 🎉 🎉  🎉